compliance/rio/src/test/java/org/eclipse/rdf4j/rio/helpers/RDFParserHelperTest.java

```diff
@@ -108,7 +108,7 @@ public final void testCreateLiteralLabelOnly()
 				valueFactory);
 
 		assertEquals(LABEL_TESTA, literal.getLabel());
-		assertFalse(literal.getLanguage().isPresent());
+		assertFalse(literal.getLanguage() != null);
```

Comment:
I did it this way because that was the simplest change I could make, and one I could easily copy-paste to multiple places. You're right that it's not really intuitive, but I figure it's "good enough".

There's no fixed date (yet) on end-of-support for Java 7, but I treat this as "as long as there's a need and we can do it without it severely impacting our primary development". 

As for merging: 1.0.x is intended to be a parallel _backport_ of the 2.0 branch. This means we will _never_ merge 1.0 into 2.0, only the other way around (or rather, we merge the master branch into the 1.0 branch). Fixes or feature development branches should also never start by branching off from the 1.0.x branch, but always either from master or from 2.0.x. I plan to document this in a bit more detail in the contribution guidelines.